build.fsx

```diff
@@ -214,6 +214,49 @@ Target "GitHubRelease" (fun _ ->
 )
 
 // --------------------------------------------------------------------------------------
+// .NET CLI and .NET Core
+
+let isDotnetCLIInstalled = Shell.Exec("dotnet", "--version") = 0
```

Comment:
use `--info`, it's better because write some diagnostics info, instead `--version` only print the version